<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreAdd</class>
 <widget class="QMainWindow" name="FenetreAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="FormulaireLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="nomRecette">
         <property name="text">
          <string>Nom recette</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nomRecetteEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="nbPersos">
       <item>
        <widget class="QLabel" name="nbPerso">
         <property name="text">
          <string>Nombre de personnes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nbPersoEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="aliments">
         <item>
          <widget class="QLabel" name="ingredient">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Ingrédient :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addIngredient">
           <property name="text">
            <string>Ajouter un aliment</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="steps">
       <item>
        <widget class="QLabel" name="step">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Etapes :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addStep">
         <property name="text">
          <string>Ajouter une étape</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titre">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>20</y>
      <width>791</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ajouter une recette</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider l'ajout de la recette</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
